template.json

```diff
@@ -31,12 +35,13 @@
       "stylelint-config-property-sort-order-smacss": "7.1.0"
     },
     "scripts": {
-      "lint": "eslint ./src --ext .ts,.tsx",
-      "lint:fix": "eslint ./src --ext .ts,.tsx --fix",
+      "lint": "eslint ./src ./cypress --ext .ts,.tsx",
+      "lint:fix": "eslint ./src ./cypress --ext .ts,.tsx --fix",
       "stylelint": "stylelint '**/*.scss'",
       "stylelint:fix": "stylelint '**/*.scss' --fix",
       "codebase:lint": "yarn lint && yarn stylelint",
-      "codebase:fix": "yarn lint:fix && yarn stylelint:fix"
+      "codebase:fix": "yarn lint:fix && yarn stylelint:fix",
+      "cypress:run": "cypress run"
```

Comment:
@noor-ahmed Can we add `"cypress:open": "cypress open"` so we can use it in development?
and also can we update the script list on `template/README.md`?

Reply:
@carryall added them to readme 3c3e7c7